server/settings.go

```diff
@@ -9,17 +11,34 @@ import (
 const (
 	settingOn  = "on"
 	settingOff = "off"
+
+	errStoreNewSettings = "Could not store new settings. Please contact your system administrator. Error: %v"
+	errConnectToJira    = "Your account is not connected to Jira. Please type `/jira connect`. %v"
+	subCommandAssignee  = "assignee"
+	subCommandMention   = "mention"
+	subCommandReporter  = "reporter"
+	subCommandWatching  = "watching"
 )
 
+func (connection *Connection) sendNotification(role string, hasNotification bool) bool {
+	if role != subCommandAssignee && role != subCommandMention && role != subCommandReporter && role != subCommandWatching {
```

Comment:
Since these aren't referring to actual subcommands here, I recommend using different names for these constants. Maybe `notificationRoleAssignee` etc.